Rubberduck.UnitTesting/UnitTesting/TestCodeGeneratorStatics.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using Rubberduck.Resources.UnitTesting;
+
+namespace Rubberduck.UnitTesting.UnitTesting
+{
+    public partial class TestCodeGenerator
```

Comment:
IMPOV, all of the elements here should be coming from a resource file, not hard-coded within a `.cs` file. (ref #4622 for an example).  While making it implement the template (ref #4549 ) may be out of scope for the PR, it should be easy enough to have a resource file contain all the needed elements, which will then aid any future changes to get it to conform to the template model, and thus make the code less hard-coded. We want to avoid writing code that basically takes a bunch of elements and concatenate/interleave them because if the content changes, the code almost invariably must change, too.

Reply:
This has to be done at run-time because the locale can be changed in the settings (this is why they're static properties instead of static fields). The difference between the code generation here and a predefined template is that it makes the localization much easier to maintain.